docs/index.rst

```diff
@@ -21,34 +21,21 @@ xCDAT (Xarray Climate Data Analysis Tools) is an extension of `xarray`_ for clim
         :link: getting-started-guide/index
         :link-type: doc
 
-        New to xCDAT? Check out the getting started guides.
-
```

Comment:
I just removed the text (I thought these were self-explanatory enough), but I don't object to short text descriptions.

Reply:
I'm okay with no text